src/components/search/results/resultsTable/ResultsTableRow/ResultsTableRow.tsx

```diff
@@ -134,11 +135,11 @@ export const ResultsTableRow: FC<ResultsTableRowProps> = observer(({ hit, index
                                         boundary: 'clippingParents',
                                     },
                                 },
-                            ]}
-                        >
+                            ]}>
                             <Paper elevation={10} className={classes.preview}>
                                 {previewLoading && <Loading />}
-                                <img
+                                <Image
```

Comment:
If we're going to use this component instead of <img>, let's make it not modify/"optimize" our images.

Optimizing wouldn't be possible anyway without the backend component, right?

It's enough that the browser has to be compatible with these - we don't want the Next Image component to bring more possibility of failure here.

https://nextjs.org/docs/pages/api-reference/components/image#unoptimized

Reply:
Right, again, forgot we removed SSR. Will change back to `<img>` and skip the ts-lint for this warning/error. Thanks for pointing this out